identify/identify.py

```diff
@@ -159,6 +159,15 @@ def parse_shebang(bytesio):
     cmd = tuple(_shebang_split(first_line.strip()))
     if cmd and cmd[0] == '/usr/bin/env':
         cmd = cmd[1:]
+    if cmd and cmd[0].endswith('nix-shell'):
+        if bytesio.read(2) != b'#!':
+            return cmd
+        second_line = bytesio.readline().decode('UTF-8')
+        cmd = tuple(_shebang_split(second_line.strip()))
```

Comment:
I believe we'll actually want to use `shlex.split` here since nix supports proper quoting

Reply:
solved in https://github.com/chriskuehl/identify/pull/130/commits/2a45f63977fb30fd2abd569f4488cd47549ce39a